Source/Csla/Serialization/Mobile/IMobileSerializer.cs

```diff
@@ -0,0 +1,39 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IMobileSerializer.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Custom serializer for a type</summary>
+//-----------------------------------------------------------------------
+using System.Text;
+using Csla.Core;
+
+namespace Csla.Serialization.Mobile
+{
+  /// <summary>
+  /// Custom serializer for a type
+  /// </summary>
+  public interface IMobileSerializer
+  {
+#if NET8_0_OR_GREATER
+    /// <summary>
+    /// Gets a value indicating whether this serializer can
+    /// serialize the specified type.
+    /// </summary>
+    /// <param name="type">Type to check</param>
+    static bool CanSerialize(Type type) => false;
```

Comment:
Yes, because I don't want to have to create an instance of the serializer if we're not going to use it. This allows detection of whether the serializer can handle the type _before_ we create the instance.